### DISCUSSION TITLE
Unable to run dockerized application following FastAPI docs

### DISCUSSION THREAD

**Discussion by pasca15** [fastapi/fastapi#6564] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import Optional

from fastapi import FastAPI

app = FastAPI()

# app.py
@app.get("/")
def read_root():
    return {"Hello": "World"}


@app.get("/items/{item_id}")
def read_item(item_id: int, q: Optional[str] = None):
    return {"item_id": item_id, "q": q}

# dockerfile
ROM python:3.7

RUN pip install fastapi uvicorn

EXPOSE 80

COPY ./app /app

CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "80"]
```

### Description

Following Deploying with Docker tutorial, I can't run the example given here https://fastapi.tiangolo.com/deployment/docker/.

The app.py and dockerfile as per example code.

Follow the same folder structure

### Operating System

Windows

### FastAPI Version

0.68.0

### Python Version

3.7

**Comment by alonme** [comment]
what is the error you are getting?

**Comment by raphaelauv** [comment]
read the doc -> https://docs.docker.com/engine/reference/builder/#expose

```shell
docker run -p 127.0.0.1:80:80/tcp myfastAPI
```

**Comment by pasca15** [comment]
> what is the error you are getting?

There was no error message. When I went to docker host and localhost, it showed Windows IIS.

**Comment by jiteshy** [comment]
@pasca15 Try using a different port than 80 e.g. 8000 in your Dockerfile. Your IIS is running on the same port 80 causing uvicorn to not start.

`CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "8000"]`

**Comment by pasca15** [comment]
Hi both, I have found it working in postman, although still not working in my browser. I'm happy with that it can work with postman.
